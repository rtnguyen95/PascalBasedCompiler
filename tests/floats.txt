float y = .1
float z = 0.11
float f = 1.
float f = 1.0
letter.uppercase