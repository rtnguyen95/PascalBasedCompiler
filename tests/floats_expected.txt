TOKENS        Lexemes

KEYWORD     = float
IDENTIFIER  = y
OPERATOR    = =
FLOAT       = .1
KEYWORD     = float
IDENTIFIER  = z
OPERATOR    = =
FLOAT       = 0.11
KEYWORD     = float
IDENTIFIER  = f
OPERATOR    = =
FLOAT       = 1.
KEYWORD     = float
IDENTIFIER  = f
OPERATOR    = =
FLOAT       = 1.0
IDENTIFIER  = letter
SEPARATOR   = .
IDENTIFIER  = uppercase